.storybook/preview-head.html

```diff
@@ -1,190 +1,5 @@
-        format('woff2'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-Regular.woff)
-        format('woff'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-Regular.ttf)
-        format('ttf');
-    font-display: swap;
-  }
-
-  @font-face {
-    font-family: GuardianTextEgyptian;
-    font-weight: 400;
-    font-style: italic;
-    src: url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-RegularItalic.woff2)
-        format('woff2'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-RegularItalic.woff)
-        format('woff'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-RegularItalic.ttf)
-        format('ttf');
-    font-display: swap;
-  }
-
-  @font-face {
-    font-family: GuardianTextSans;
-    font-weight: 700;
-    src: url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-Bold.woff2)
-        format('woff2'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-Bold.woff)
-        format('woff'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-Bold.ttf)
-        format('ttf');
-    font-display: swap;
-  }
-
-  @font-face {
-    font-family: GuardianTextSans;
-    font-weight: 400;
-    src: url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-Regular.woff2)
-        format('woff2'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-Regular.woff)
-        format('woff'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-Regular.ttf)
-        format('ttf');
-    font-display: swap;
-  }
-
-  @font-face {
-    font-family: GuardianTextSans;
-    font-weight: 400;
-    font-style: italic;
-    src: url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-RegularItalic.woff2)
-        format('woff2'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-RegularItalic.woff)
-        format('woff'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-RegularItalic.ttf)
-        format('ttf');
-    font-display: swap;
-  }
-</style>
+<script>
+  // fix for chromatic/storybook failing to render a component
+  // as the window.guardian object was not defined
+  window.guardian = {};
```

Comment:
I guess we don't need the font declarations and other nonsense any more?

Reply:
Well that's all still there, just under the `GlobalStyles` component now in preview.js

Reply:
Oh, duh. :P